packages/analytics-js-integrations/__tests__/integrations/Amplitude/amplitude.test.js

```diff
@@ -62,4 +76,16 @@ describe('Amplitude init tests', () => {
     // eslint-disable-next-line no-underscore-dangle
     expect(typeof window.amplitude).toBe('object');
   });
+  test('Testing init call of Amplitude with EU residency set', () => {
+    const amplitude = new Amplitude(
+      amplitudeEUConfig,
+      { loglevel: 'debug', loadIntegration: true },
+      destinationInfo,
+    );
+    amplitude.init();
+    // eslint-disable-next-line no-underscore-dangle
+    expect(typeof window.amplitude).toBe('object');
+    // eslint-disable-next-line no-underscore-dangle
+    expect(amplitudeEUConfig.residencyServer).toBe('EU');
```

Comment:
there's no use of this assertion. 
What I meant was to assert whether EU endpoint is being set in window.amplitude object. If it's not feasible, fine, we need to cover in e2e test suite.

Reply:
Just look if we can get `window.amplitude.serverZone`, this can be explored later too

Reply:
Couldn't get this: Can check later

`window.amplitude.serverZone`

Reply:
handled